components/nextra/Flexsearch.tsx

```diff
@@ -36,38 +36,30 @@ import { SearchResult } from './types';
 // Diff: Inlined definitions
 export const DEFAULT_LOCALE = 'en-US';
 
-type SectionIndex = FlexSearch.Document<
```

Comment:
These are no longer generic, so I stripped them but left the types in place in case they reinstate this behavior later